connect-example.py

```diff
@@ -1,14 +1,63 @@
 from pymongo import MongoClient
 from dotenv import load_dotenv
+from faker import Faker
+from faker.providers import DynamicProvider
 import os
 
+job_provider = DynamicProvider(
+    provider_name="job", elements=["designer", "accounting", "developer", "marketing"]
+)
+age_provider = DynamicProvider(provider_name="age", elements=["20", "25", "30", "40"])
+database_provider = DynamicProvider(
+    provider_name="database",
+    elements=[
+        "text",
+        "hello",
+        "payments",
+        "customers",
+        "posts",
+        "database",
+        "databases",
+    ],
+)
+fake = Faker()
+fake.add_provider(job_provider)
+fake.add_provider(age_provider)
+fake.add_provider(database_provider)
+
 load_dotenv(verbose=True)  # take environment variables from .env.
 
 MONGO_DATABASE_URI = os.getenv("MONGO_DATABASE_URI")
+MONGO_SLAVE_URI = os.getenv("MONGO_DATABASE_URI")
```

Comment:
Interesting approach 

Ther's no need to specify the secondary nodes  (and we don't know what they are, since primary/secondary may change over time)

Instead, update the connection string `MONGO_DATABASE_URI` to include all nodes in the cluser

e.g. 

```
MONGO_DATABASE_URI=mongodb://[username:password@]host1[:port1][,...hostN[:portN]][/[defaultauthdb][?options]]

e.g.
MONGO_DATABASE_URI=mongodb://fred:secret@<ip-address>,fred:secret@<ip-address-two>,fred:secret@<ip-address-three>[/[defaultauthdb][?options]]
```

See also
https://stackoverflow.com/a/26091582